libipl/p10/ipl0.C

```diff
@@ -277,6 +277,43 @@ static bool update_genesis_hwas_state(void)
 	return true;
 }
 
+/**
+ * @brief Wrapper function to execute continue mpipl on the pib
+ *
+ * @param[in] pib pib target to operate on
+ *
+ * return ipl error type enum
+ */
+static ipl_error_type  ipl_sbe_mpipl_continue(struct pdbg_target *pib)
+{
+	ipl_log(IPL_INFO, "Enter: ipl_sbe_mpipl_continue(%s)", pdbg_target_path(pib));
```

Comment:
Usually any messages should have a prefix that identifies class of messages.  Instead of "Enter : foobar",  I would prefer "foobar: Enter", where in this case foobar represents the context (e.g. function name).

Reply:
Done